tests/test_schema.py

```diff
@@ -609,6 +610,43 @@ def test_many_id_errors(self):
         assert id_err["source"]["pointer"] == "/data/1/id"
         assert id_err["detail"] == "Not a valid string."
 
+    def test_nested_fields_error(self):
+        def validate_number(data):
+            if data < 10:
+                raise ValidationError("Invalid Number.")
+
+        class ThirdLevel(ma.Schema):
+            number = fields.Int(required=True, validate=validate_number)
```

Comment:
You can use `marshmallow.validate.Range`:

```python
number = fields.Int(required=True, validate=Range(min=10))
```

Reply:
I am just not sure it would hit the problem I want to fix but I will take a look on that to.

Reply:
Much better indeed. Thank you.